tests/helpers/stubs.py

```diff
@@ -584,6 +585,9 @@ def page_title(self, i):
     def on_tab_close_requested(self, idx):
         del self.tabs[idx]
 
+    def tabBar(self):
+        return QTabBar(self._qtabbar)
```

Comment:
Why call the `QTabBar` constructor again here (which probably just copies the existing one)?